pyproject.toml

```diff
@@ -102,6 +102,8 @@ doc = [
     "jupytext",
     "myst-nb",
     "pooch",
+    "jupyterlite-sphinx>=0.12.0",
```

Comment:
Please also add these in the environment.yml and also in the doc about the toolchain.

Reply:
Thanks. I missed this.